src/nspawn/nspawn.c

```diff
@@ -195,6 +195,7 @@ static const char *arg_container_service_name = "systemd-nspawn";
 static bool arg_notify_ready = false;
 static bool arg_use_cgns = true;
 static unsigned long arg_clone_ns_flags = CLONE_NEWIPC|CLONE_NEWPID|CLONE_NEWUTS;
+static MountSettingsMask mount_settings = 0;
```

Comment:
These global, static variables that directly relate to cmdline arguments (or in some case, including this one, to env vars), should all be prefixed with the "arg_" prefix, to indicate that their are process arguments instead of internal state. We followed that pretty strictly so far in our tree, and should do so here, I think